sdk/node/src/crypto.ts

```diff
@@ -0,0 +1,645 @@
+/**
+ * Copyright 2016 IBM
```

Comment:
Please align the copyright with the template [here](https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt).